charts/hlf-k8s/README.md

```diff
@@ -120,15 +126,19 @@ Install a chaincode on a peer using the following values.
 On a peer:
 
 ```yaml
-chaincodes:
-  - name: mycc     # Chaincode name
-    version: "1.0" # Chaincode version
-    # Chaincode source code archive URL
-    src: https://github.com/SubstraFoundation/substra-chaincode/archive/0.0.2.tar.gz
+appChannels:
+    chaincodes:
```

Comment:
```suggestion
  - channelName: mychannel
    chaincodes:
```
Showing the fact that a chaincode lives inside a channel could be a good idea, what do you think ?

Reply:
Yes !